File: VisitAppl/app/src/main/java/kz/visitapp2/RegistrationPageClasses/UserRegPage.java
package kz.visitapp2.RegistrationPageClasses;

import android.content.Intent;
import android.os.Build;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.view.Window;
import android.view.WindowManager;
import android.widget.ImageButton;

import kz.visitapp2.InterestsPageClasses.InterestsPage;
import kz.visitapp2.R;

public class UserRegPage extends AppCompatActivity {
    ImageButton skip;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
            Window w = getWindow();
            w.addFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
            w.addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
        }
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_user_reg_page);

        skip = findViewById(R.id.facebook_btn);
        skip.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(UserRegPage.this, InterestsPage.class);
                startActivity(intent);
                overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
            }
        });


    }

}

File: VisitAppl/app/src/main/java/kz/visitapp2/Activities/RegistrationPageActivities/UserRegPage.kt
package kz.visitapp2.Activities.RegistrationPageActivities

import android.content.Intent
import android.os.Build
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.WindowManager
import android.widget.ImageButton

import kz.visitapp2.Activities.InterestsPageActivities.InterestsPage
import kz.visitapp2.R

class UserRegPage : AppCompatActivity() {
    internal lateinit var skip: ImageButton
    override fun onCreate(savedInstanceState: Bundle?) {
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
            val w = window
            w.addFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS)
            w.addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS)
        }
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_user_reg_page)

        skip = findViewById(R.id.facebook_btn)
        skip.setOnClickListener {
            val intent = Intent(this@UserRegPage, InterestsPage::class.java)
            startActivity(intent)
            overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
        }


    }

    override fun onBackPressed() {
        super.onBackPressed()
        overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out)
    }

}
